modules/lotamePanoramaIdSystem.js

```diff
@@ -189,7 +189,15 @@ export const lotamePanoramaIdSubmodule = {
       if (consentData && utils.isBoolean(consentData.gdprApplies)) {
         queryParams.gdpr_applies = consentData.gdprApplies;
         if (consentData.gdprApplies) {
-          queryParams.gdpr_consent = consentData.consentString;
+          let consentString = consentData.consentString;
+          // If no consent string in the data, try to read it from 1st party cookies
+          if (!consentString) {
+            consentString = getFromStorage('eupubconsent-v2');
+          }
+          if (!consentString) {
+            consentString = getFromStorage('euconsent-v2');
+          }
+          queryParams.gdpr_consent = consentString;
```

Comment:
we shouldn't add a query param unless it actually has a value here

Reply:
Done! Or, not done? You know what I mean...